content/en/docs/tutorials/github.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Setup GitHub Actions"
+subtitle: "Setup GitHub Actions for Continuous Integration"
+weight: 5
+draft: false
+description: >
+    A tutorial on how to set up Score with GitHub Actions.
+---
+
+You can use `score-humanitec` in your GitHub Action workflows, for example, when you make a code change, and you want the Humanitec Platform Orchestrator to deploy your code to a new environment.
+
+The following is a guide to setting up the Score implementation CLI in your GitHub Action workflows.
+
+For the Score GitHub Action, see [Setup Score](https://github.com/score-spec/setup-score)
+
+## Usage
+
+To use the Score GitHub Action, add the following step to your workflow:
```

Comment:
"workflow" sounds a little vague, I wouldn't necessarily know where to add the snippet below. We could link to the GitHub actions docs here or so? https://docs.github.com/en/actions/using-workflows/about-workflows